packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1199,14 +1198,16 @@ If more than 100 groups match the filter, then the request fails. Expect that th
 For more information about configuring an app for OpenID Connect, including group claims, see [Create a client application](/docs/guides/federate-with-oidc/create-client-app/).
 * **Important:** Scope-dependent claims are returned differently depending on the values in `response_type` and the scopes requested:
 
-| Response Type               | Claims Returned in ID Token                                                                          | Claims Returned from the Userinfo Endpoint  |
-| :-------------------------- | :--------------------------------------------------------------------------------------------------- | :------------------------------------------ |
-| `code `                     | N/A                                                                                                  | N/A                                         |
-| `token`                     | N/A                                                                                                  | N/A                                         |
-| `id_token`                  | Claims associated with requested scopes.                                                             | N/A                                         |
-| `id_token` `code `          | Claims associated with requested scopes.                                                             | N/A                                         |
-| `id_token` `token`          | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested   | Claims associated with the requested scopes |
-| `code` `id_token` `token`   | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested   | Claims associated with the requested scopes |
+  | Response Type             | Claims Returned in ID Token from Default AS                                                        | Claims Returned in ID Token from Custom AS | Claims Returned from Userinfo Endpoint     |
+  | :-------------------------| :------------------------------------------------------------------------------------------------- | :----------------------------------------- | :----------------------------------------- |
+  | `code `                   | N/A                                                                                                | N/A                                        | N/A                                        |
+  | `token`                   | N/A                                                                                                | N/A                                        | N/A                                        |
+  | `id_token`                | Claims associated with requested scopes.                                                           | N/A                                        | N/A                                        |
```

Comment:
N/A for custom AS should be the same as Org AS `Claims associated with requested scopes.`